src/pextlib1.0/tests/curl.tcl

```diff
@@ -32,8 +32,9 @@ proc main {pextlibname} {
     #curl fetch -u "I accept www.opensource.org/licenses/cpl:." http://www.research.att.com/~gsf/download/tgz/sfio.2005-02-01.tgz $tempfile
     #test {[md5 file $tempfile] == "48f45c7c77c23ab0ccca48c22b3870de"}
 
-    curl fetch --enable-compression https://www.whatsmyip.org/http-compression-test/ $tempfile
-    grepper $tempfile {gz_yes}
+    # CNielsen: Temporarily disable compressed fetch, as failing after 30-second timeout
+    #curl fetch --enable-compression https://www.whatsmyip.org/http-compression-test/ $tempfile
+    #grepper $tempfile {gz_yes}
```

Comment:
This doesn’t seem directly related to the debug level changes, or am I missing something ?

Reply:
This was necessary back a month or so ago, as the test consistently failed due to timeout. (It wasn't intended to be kept long-term though.)

Ultimately it will be reverted in this PR, or changed to use a different testing endpoint if necessary. (The latter would be based on guidance from Josh and/or Ryan though.)

Reply:
hm, @jmroot @ryandesign any guidance?